Applications/Custom/mae_loss.cpp

```diff
@@ -12,8 +12,57 @@
  */
 #include "mae_loss.h"
 
+#include <cmath>
+
+#include <tensor.h>
+
+constexpr const float EPSILON_ = 1e-7;
 namespace custom {
-const std::string MaeLossLayer::type = "mae_loss";
+
```

Comment:
when the developer use their own layer in API ( not ini file ), How they call it? do they have to use custom namespace as defined here?

Reply:
namspace is up to the developer.

To create...
1. Register *.so and createLayer('custom_type') (similar to nnstreamer sub-plugin) 
2. Directly creates a instance out of the class.

Please refer to the details on the following PR :) https://github.com/nnstreamer/nntrainer/pull/1320

Reply:
thanks.